### ISSUE TITLE
Tab jumping into position

### ISSUE THREAD

**Issue by jjdelc** [autonome/Always-Right#16]
When opening a new tab, I can see it get open on the default spot and then it jumps to the right of my active tab.

**Comment by autonome** [comment]
Yep, unfortunately it's a limitation of the Tabs WebExtensions API. The bug
is being tracked here: https://bugzilla.mozilla.org/show_bug.cgi?id=1387372

**Comment by peteruithoven** [comment]
Looks like this was fixed in the following commit? 
https://github.com/autonome/Always-Right/commit/83ad05b14a3022af4182446c9cdb29336ac6d985
(relevant: https://github.com/autonome/Always-Right/issues/22)

And this it seems to work great in Firefox nightly 62, tabs are opened directly after the current instead of first being opened at the end and then moved.
